binding/python/docs/EXPERIMENTS.md

```diff
@@ -0,0 +1,60 @@
+# Experiments
+
+Here is the result of running [Deep_Residual_Learning_CIFAR-10](../examples/theano/lasagne/Deep_Residual_Learning_CIFAR-10.py)
+
+## Task Description
+Perform CIFAR-10 classification with residual networks implementation based on Lasagne.
+
+## Hardware
+|||
+| -------- |:--------:|
+|Hosts|1|
+|GPU|GeForce GTX TITAN X * 4|
+|CPU|Intel(R) Core(TM) i7-5960X CPU @ 3.00GHz  * 1|
+|Memory| 128GB |
+
+
+## Theano settings
+Here is the content of `~/.theanorc`
+```
+[global]
+device = gpu
+floatX = float32
+
+[cuda]
+root = /usr/local/cuda-7.5/
+
+[lib]
+cnmem = 1
+```
+
+## About the Model
+|||
+| :---- | -----: |
+|Total epoch|82|
+|Batch size|128|
+|Depth|32|
+|Learning rate change schedule|Initialized as 0.1, Changed to 0.01 from epoch 41, Changed to 0.001 from epoch 61|
+|number of parameters in model|    464,154|
+
+
+Clarification
+- An epoch represents all the processes divide all the data equally and go through them once together.
+- A barrier is used at the end of every epoch.
+- This experiment didn't use warm start in ASGD.
+- The time to load the data is not considered in the time of the experiment.
+
+
+# The results
+4 experiments have been run. The configuration and the results of each configuration are listed below.
```

Comment:
The results of 4 experiments with different configurations are shown as following.